mitodl/open-discussions
# Create one run per program

Based on #2265, which should be reviewed first

#### Pre-Flight checklist

#### What are the relevant tickets?
Closes #2264
Closes #2254

#### What's this PR do?
- Removes `prices` from `Program` model
- Creates 1 `LearningResourceRun` per `Program`, containing prices, dates, etc.

#### How should this be manually tested?
- Rerun import tasks for programs:
   ```
    get_xpro_data()
    get_micromasters_data()
    ```
- Go to `courses/search` and select `Programs` - results should still display correctly, plus the 'Availability' and 'Cost' facets should still display with checkbox options, which should work.
- Click on some programs, the drawer should still open and display the correct info.
